python/pysrc/kaskada/_timestream.py

```diff
@@ -880,6 +893,84 @@ def record(self, fields: Callable[[Timestream], Mapping[str, Arg]]) -> Timestrea
         """
         return record(fields(self))
 
+    def round(self) -> Timestream:
+        """Return a Timestream with all values rounded to the nearest integer.
+
+        Returns:
+            A Timestream of the same type as `self`. The result contains `null`
+              if the value was `null` at that position. Otherwise, it contains
+              the result of rounding the value to the nearest integer.
+
+        Notes:
+            This method may be applied to any numeric type. For anything other
+              than `float32` and `float64` it has no affect since the values
+              are already integers.
+
+        See Also:
+            `ceil` and `floor`
+        """
+        return Timestream._call("round", self)
+
+    def sqrt(self) -> Timestream:
+        """Return a Timestream with the square root of all values.
+
+        Returns:
+            A Timestream of type `float64`. The result contains `null`
+              if the value was `null` at that row. Otherwise the row
+              contains the square root the value.
+        """
+        return Timestream._call("sqrt", self)
+
+    def upper(self) -> Timestream:
+        """Return a Timestream with all values converted to upper case."""
+        return Timestream._call("upper", self)
+
+    def zip_max(self, rhs: Union[Timestream, Literal]) -> Timestream:
+        """Return a Timestream with the maximum of `self` and `rhs`
+
+        Args:
+            rhs: The Timestream or literal value to compare to this.
+
+        Returns:
+            A numeric Timestream of the promoted type. Each row
+              contains the value from `self` if `self` is greater
+              than `rhs`, otherwise it contains `rhs`. If `self` or
+              `rhs` is `NaN` or `null`, then `rhs` will be returned.
+
+        Notes:
+            Both `self` and `rhs` are promoted to a compatible ordered
+              type following the docs:data-model#numeric-type-coercion-table[numeric type coercion rules].
+
+        See Also:
+            This returns the maximum of two values. See the
```

Comment:
Confirm this links to `max`. Otherwise, you may need:

```
See Also:
  max: ...
```

or something like that.

Reply:
`:func:\`max\``  works.  

I also added see-also on `max` and `min`, pointing to `greatest` and `least`